andela/ninjas-ah-backend
# #165412916 create user profile

#### What does this PR do?
Add an endpoint that allows users to update their profile
#### Description of Task to be completed?
Have the following endpoint accessible to an authenticated user
- `PUT /api/v1/users`
#### How should this be manually tested?
- After cloning the repo, cd into it and type `npm test`
- Using Postman, firstly make sure you signup then login to get the access token, secondly make sure you pass the access token in the header with the following key name: `access-token` to be able to edit the profile
#### What are the relevant pivotal tracker stories?
[165412916](https://www.pivotaltracker.com/story/show/165412916)
